### PR TITLE
Update S3 documentation 

### PR DESCRIPTION
**Pull request by colleenkerr**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @internetisaiah as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] If the documentation involves a 1) paid SKU, 2) a third party, 3) SMS, 4) AI, or 5) privacy, ensure that Ana Teresa Serafino on the Legal team has signed off.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @josh-mccrowell-braze or @internetisaiah for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@josh-mccrowell-braze</td>
    <td>Monolith Deployments<br>Quality Infrastructure<br>Platform Infrastructure<br>Datalake<br>SDKs<br>Currents</td>
  </tr>
  <tr>
    <td>@internetisaiah</td>
    <td>Developer Guide<br>Technology partners<br>Channels</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Intelligence<br>In-App Messages<br>Channels<br>Frontend Infrastructure & Experience (FIX)</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Ingestion<br>Core Objects<br>Core Messaging<br>Messaging Experience<br>Message Components<br>Email (Composition and Infrastructure) (tag @rachel-feinberg for Liquid use cases)</td>
  </tr>
  <tr>
    <td>@rachel-feinberg</td>
    <td>Customer Lifecycle, Identity and Permissions<br>SMS<br>User Targeting<br>Reporting</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/file_storage_integrations.md [hunk 1]
```diff
@@ -42,9 +42,19 @@ Take note of the region you’ve created the bucket in, as you will create an SQ
...
+{% alert important %}
+Note: Be sure to create this SQS in the same region you created the bucket in. {% endalert %}
+
+Use default configuration settings until you reach the access policy step. When setting up the access policy, choose Advanced options. 
+
+Be sure to take note of the ARN and the URL of the SQS as you’ll be using it in your changes, and later on when making the IAM Policy and CDI Integration. 
+<br><br>![]({% image_buster /assets/img/cloud_ingestion/s3_ARN.png %})
+<br><br>
+
+Append the following statement to the queue's access policy, being careful to replace `YOUR-BUCKET-NAME-HERE` with your bucket name, and `YOUR-SQS-ARN` with your SQS queue ARN, and `YOUR-AWS-ACCOUNT-ID` with your AWS account ID..
```
Comment (josh-mccrowell-braze) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/file_storage_integrations.md:56]:
```suggestion
Append the following statement to the queue's access policy, being careful to replace `YOUR-BUCKET-NAME-HERE` with your bucket name, and `YOUR-SQS-ARN` with your SQS queue ARN, and `YOUR-AWS-ACCOUNT-ID` with your AWS account ID. 
```

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/file_storage_integrations.md [hunk 2]
```diff
@@ -42,9 +42,19 @@ Take note of the region you’ve created the bucket in, as you will create an SQ
...
 ### Step 2: Create SQS resources
 
-Create an SQS queue in the same region you created the S3 bucket. This queue will be used to track when objects are added to the bucket(s) you’ve created.
+Create an SQS queue to track when objects are added to the bucket you’ve created.
 
-You can create this queue with default configuration settings until you reach the access policy step. When setting up the access policy, choose Advanced options. Append the following statement to the queue's access policy: 
+{% alert important %}
+Note: Be sure to create this SQS in the same region you created the bucket in. {% endalert %}
+
+Use default configuration settings until you reach the access policy step. When setting up the access policy, choose Advanced options.
```
Comment (josh-mccrowell-braze) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/file_storage_integrations.md:50]:
```suggestion
Use default configuration settings until you reach the access policy step. When setting up the access policy, choose **Advanced options**. 
```

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/file_storage_integrations.md [hunk 3]
```diff
@@ -156,3 +166,61 @@ To complete the setup on AWS, you will create an IAM role and attach the IAM pol
...
+Every line in your source file must contain valid JSON, or the file will be skipped. {% endalert %}
+{% endtab %}
+{% tab CSV Attributes %}
+``` csv  
+external_id,payload
+s3-qa-load-0-d0daa196-cdf5-4a69-84ae-4797303aee75,"{""name"": ""SNXIM"", ""age"": 54, ""subscriber"": true, ""retention"": {""previous_purchases"": 19, ""vip"": true}, ""last_visit"": ""2023-08-08T16:03:26.598806""}"
+s3-qa-load-1-d0daa196-cdf5-4a69-84ae-4797303aee75,"{""name"": ""0J747"", ""age"": 73, ""subscriber"": false, ""retention"": {""previous_purchases"": 22, ""vip"": false}, ""last_visit"": ""2023-08-08T16:03:26.598816""}"
+s3-qa-load-2-d0daa196-cdf5-4a69-84ae-4797303aee75,"{""name"": ""EP1U0"", ""age"": 99, ""subscriber"": false, ""retention"": {""previous_purchases"": 23, ""vip"": false}, ""last_visit"": ""2023-08-08T16:03:26.598822""}"
+```  
+{% endtabs %}
```
Comment (internetisaiah) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/file_storage_integrations.md:224]: Looks like this needs a:

```
{% endtab %}
{% endtabs%}
```

Going to push a commit to fix this now 👍🏽

File: _docs/_user_guide/data_and_analytics/cloud_ingestion/file_storage_integrations.md [hunk 4]
```diff
@@ -42,9 +42,19 @@ Take note of the region you’ve created the bucket in, as you will create an SQ
...
 ### Step 2: Create SQS resources
 
-Create an SQS queue in the same region you created the S3 bucket. This queue will be used to track when objects are added to the bucket(s) you’ve created.
+Create an SQS queue to track when objects are added to the bucket you’ve created.
 
-You can create this queue with default configuration settings until you reach the access policy step. When setting up the access policy, choose Advanced options. Append the following statement to the queue's access policy: 
+{% alert important %}
+Note: Be sure to create this SQS in the same region you created the bucket in. {% endalert %}
+
+Use default configuration settings until you reach the access policy step. When setting up the access policy, choose **Advanced options**.
```
Comment (josh-mccrowell-braze) [inline _docs/_user_guide/data_and_analytics/cloud_ingestion/file_storage_integrations.md:50]: @colleenkerr Is this duplicative with "The default settings are:" (line 48) from step 1? 

In either case, what is the access policy step? Step 4?

**Comment by josh-mccrowell-braze** [comment]
@colleenkerr : I added a definitions section at the top. I also had a single question about the workflow. Would you please re-review? Afterwards, I'll finalize and merge. Thanks!

**Review by colleenkerr — COMMENTED** [review]
LGTM

**Review by jayhahn — APPROVED** [review]

**Pull request merged**